shopify/data/data_schema.graphql

```diff
@@ -1,18 +1,38 @@
 ## Represents the address of a location. used for billing and shipping addresses.
-type Address {
+type MailingAddress {
   address1: String
   address2: String
   city: String
-  country_code: String
+  company: String
   country: String
-  email: String
+  countryCodeV2: String
+  firstName: String
+  lastName: String
+  name: String
   phone: String
-  province_code: String
   province: String
-  state: String
+  provinceCode: String
   zip: String
 }
 
+type FulfillmentOriginAddress {
+  address1: String
+  address2: String
+  city: String
+  countryCode: String!
+  provinceCode: String
+  zip: String
+}
+
+type Money {
+  amount: Currency
+  currencyCode: String
+}
+
+type MoneySet {
+  shop: Money
```

Comment:
same comment about presentment Money. we should include it in our responses and schema